etc/skydive.yml.default

```diff
@@ -105,12 +107,15 @@ agent:
   # X509_servername: domain.com
   #
   http:
-    # log the HTTP client request and response (to log level DEBUG)
-    # debug: false
     # define the Cookie HTTP Request Header
     # cookie: 
     #   <name1>: <value1>
     #   <name2>: <value2>
+    # rest:
+    #   log the HTTP client request and response (to log level DEBUG)
```

Comment:
Not sure that this is related to the agent, maybe we should move the http section outside of the agent section.

Reply:
agree, have moved this into 'http.rest.*'